openraft/src/replication/mod.rs

```diff
@@ -219,14 +219,14 @@ impl<C: RaftTypeConfig, N: RaftNetworkFactory<C>, S: RaftStorage<C>> Replication
             log_reader,
             config,
             target_repl_state: TargetReplState::LineRate,
-            last_log_id: last_log,
             committed,
             matched: None,
             max_possible_matched_index: last_log.index(),
             raft_core_tx,
             repl_rx,
             heartbeat: interval(heartbeat_timeout),
             install_snapshot_timeout,
+            need_to_replicate: false,
```

Comment:
What about initializing it with `true`?

When a replication stream is just created, it has no idea about the state of the remote peer.
The first `send_append_entries` will update the state.

Although the implementation may always send an initial RPC to the remote peer no matter what the value of this field is, a `true` will give a reader a consistent view of the code base.